src/api/todos.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1767;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const addTodo = (todo: Omit<Todo, 'id'>) => {
+  return client.post(`/todos`, { ...todo, userId: USER_ID });
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const changeTodoParams = (
+  todoId: number,
+  param: string,
+  paramValue: Todo['title'] | Todo['completed'],
+) => {
+  return client.patch(`/todos/${todoId}`, { [param]: paramValue });
+};
```

Comment:
you can pass an object with all the properties into the `.patch()` directly
```suggestion
export const changeTodoParams = (
  todoId: Todo['id'],
 newData: Partial<Todo>,
) => {
  return client.patch(`/todos/${todoId}`, newData);
};
```